modules/lightning_features/lightning_media/src/BundleResolverBase.php

```diff
@@ -13,6 +13,8 @@
  */
 abstract class BundleResolverBase extends PluginBase implements BundleResolverInterface, ContainerFactoryPluginInterface {
 
+  use SourceFieldTrait;
```

Comment:
Wasn't it essentially already part of `BundleResolverBase` though? It had a method `getSourceField` which I removed in favor of using the trait since it was mostly code duplication.